tests/rust/src/ramble.rs

```diff
@@ -0,0 +1,248 @@
+///////////////////////////////////////////////
+// This file was generated using Ramble.
+///////////////////////////////////////////////
```

Comment:
This file will be deleted on merge. Its created as an example of what the generated file looks like, rather than having to parse handlebar files.